airflow/dag_processing/manager.py

```diff
@@ -99,6 +102,13 @@ class DagFileStat:
 log = logging.getLogger("airflow.processor_manager")
 
 
+class DagFilePath(NamedTuple):
```

Comment:
Should this be named something like DagFileInfo instead?

Reply:
Yes, probably. I know @dstandish had a suggestion too, but I can't recall what it was?

Reply:
Went with that name for now :)